### ISSUE TITLE
navigateRegions doesn't work from the editable blocks

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#5847]
`navigateRegions` is a keyboard-friendly feature that allows to navigate through the main Gutenberg areas with a keyboard shortcut. Pressing `Ctrl + backtick` focus is moved and cycles through the top toolbar, the editing area, and the sidebar. `Shift + Ctrl + backtick` cycles in the opposite direction.

Inspired by a similar feature in Slack, it has been brilliantly implemented and it's a great feature for keyboard users.

Unfortunately, seems to work only when focus is not inside an editable block, which kinda defects its purpose. At any moment, users should be able to use `Ctrl + backtick` to jump through the editor main sections, evne (an especially) when they're typing in an editable block.

To reproduce:
- using the keyboard, navigate to a non-editable block e.g. an image block
- press `Ctrl + backtick` and observe focus cycles through the 3 main editor sections
- tab into a paragraph, in the text area
- press `Ctrl + backtick`
- nothing happens

Note: honestly I'm not sure if `navigateRegions` has ever worked from editable blocks, but it's of fundamental importance to make it work from anywhere.

**Comment by afercia** [comment]
As far as I see, all that's needed here is to use `bindGlobal` for the `Mousetrap` shortcuts. PR incoming.
